runbot_travis2docker/demo/runbot_repo_demo.xml

```diff
@@ -3,7 +3,7 @@
     <data>
 
         <record id="runbot_repo_demo1" model="runbot.repo">
-            <field name="name">github.com/vauxoo-dev/fast-travis-test.git</field>
+            <field name="name">https://github.com/vauxoo-dev/fast-travis-test.git</field>
```

Comment:
I don't think the code is working if this data didn't work. The repo name didn't start with `https://` or `git@`, so shouldn't the `https` have been [appended automatically](https://github.com/OCA/runbot-addons/pull/106/commits/0dc83f9efa36ff9602272d70d4cde76f19704b40#diff-8cc021a18c100ed3cee5c1d675c8e723R175)?